src/hres_eccb.py

```diff
@@ -21,6 +21,7 @@
 
 import replace_landsurface_with_ERA5land_IC
 import replace_landsurface_with_BARRA2R_IC
+import replace_landsurface_with_FF_IC
```

Comment:
This import isn't used in this file, I don't think.

Reply:
Yes you are correct Tommy.